### DISCUSSION TITLE
Feature Req: Link on the left side of the Heading

### DISCUSSION THREAD

**Discussion by d5364t54ytfr4** [django-wiki/django-wiki#1248] [Q&A]
This project shows links for specific parts in the TOC, but it'd be better if there was a link on the left side of every heading. For example Github wiki(for this project too) shows links on every heading when you place your mouse pointer on the left side of the heading. You can them right click and copy the link and paste it anywhere.

Like this link, but on the left side of every heading
https://demo.django-wiki.org/#wiki-toc-code-blocks

**Comment by benjaoming** [comment]
Hi @4getter :wave: 

Do you mean an anchor link? Like the ones that typically have an icon like :paperclip: or ¶ ?

**Comment by oscarmcm** [comment]
Hi @4getter 

You can also override the template used at your project level in order to add this kind of customization level.
